SPECS-EXTENDED/ucx/ucx.spec

```diff
@@ -0,0 +1,431 @@
+%{!?configure_options: %global configure_options %{nil}}
+%bcond_without cma
+%bcond_with    cuda
+%bcond_with    gdrcopy
+%bcond_without ib
+%bcond_without rdmacm
+%bcond_with    rocm
+%bcond_with    ugni
+%bcond_with    xpmem
+%bcond_with    vfs
+%bcond_with    mad
+%bcond_without mlx5
+
+Summary:        UCX is a communication library implementing high-performance messaging
+Name:           ucx
```

Comment:
What happens to the ucx package out of SPECS?

Reply:
@christopherco `ucx` version 1.18 has new features that MOFED requires. `ucx` version 1.18 is still not published as stable release. I've been told that Azure Linux shiproom gave us green light to add this rc version to extended repo and when the stable version is out we should be able to promote it to the core.